service/src/main/kotlin/fi/espoo/evaka/application/notes/ApplicationNoteQueries.kt

```diff
@@ -5,11 +5,13 @@
 package fi.espoo.evaka.application.notes
 
 import fi.espoo.evaka.application.ApplicationNote
+import fi.espoo.evaka.shared.ApplicationId
+import fi.espoo.evaka.shared.ApplicationNoteId
 import fi.espoo.evaka.shared.db.Database
 import org.jdbi.v3.core.kotlin.mapTo
 import java.util.UUID
 
-fun Database.Read.getApplicationNoteCreatedBy(id: UUID): UUID {
+fun Database.Read.getApplicationNoteCreatedBy(id: ApplicationNoteId): UUID {
```

Comment:
Fully using `PersonId` / `ChildId` (alias for PersonId) / `EmployeeId` is left for a later pull request, because person and employee are sometimes mixed and now that they are distinct types, all that code needs to be refactored somehow.